src/styles/index.css

```diff
@@ -0,0 +1,393 @@
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: calc(100px * 7 + 1px * 6 + (10px));
+  display: grid;
+  grid-template-columns: repeat(7, 100px);
+  gap: 1px;
+  padding: 10px;
+  margin-left: 10px;
```

Comment:
The `margin-left` property here is not symmetrical with the rest of the margins. If you want to center the calendar, you should consider using `margin: auto` or other centering techniques that are consistent in all directions. This will improve code readability and maintain the centering regardless of the calendar's width.